<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>351</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Setup</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>191</width>
        <height>181</height>
       </rect>
      </property>
      <property name="title">
       <string>Board Setting</string>
      </property>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
      <widget class="QComboBox" name="PortComboBox">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>20</y>
         <width>94</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>64</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Baudrate</string>
       </property>
      </widget>
      <widget class="QComboBox" name="BaudrateComboBox">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>50</y>
         <width>94</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="ConnectPushBtn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>161</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>64</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Status :</string>
       </property>
      </widget>
      <widget class="QLabel" name="StatusLabel">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>150</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Disconnected</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>64</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Servo ID</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ServoIDLineEdit">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>80</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="SetupExitPushBtn">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>460</y>
        <width>61</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QLabel" name="SaveStatusLabel">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>460</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Run</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actiona">
   <property name="text">
    <string>a</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
